Shared/Transforms/Variable.cs

```diff
@@ -0,0 +1,334 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace Coudinary.NetCoreShared.Transforms
```

Comment:
Each of the following classed Variable, expression and PredefinedVariable should reside in a designated file.

Reply:
Agreed, on it.